resources/views/tasks/preview.blade.php

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta http-equiv="Content-Security-Policy" content="script-src * 'unsafe-inline' 'unsafe-eval'; object-src 'self';">
+    <meta http-equiv="Content-Security-Policy" content="content="script-src * 'unsafe-inline' 'unsafe-eval'; object-src 'self'; worker-src 'self' blob:;">
```

Comment:
@danloa I think there's a typo error; the word `content` appears twice. Could you check it?